src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -418,18 +418,25 @@ def _question_name_to_id(self) -> Dict[str, "UUID"]:
 
     def __repr__(self) -> str:
         """Returns a string representation of the dataset."""
-        indent = "   "
-        return (
```

Comment:
@kursathalat we should just use the old default formatting as is shown for the `FeedbackDataset` but add `metadata_properties` to it.

Additionally, we need to add some tests.